sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -402,13 +402,14 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
 
   test("Invalid interval term should throw AnalysisException") {
     val sql1 = "select interval '42-32' year to month"
-    val value1 = "Error parsing interval year-month string: " +
-      "requirement failed: month 32 outside range [0, 11]"
     val fragment1 = "'42-32' year to month"
     checkError(
       exception = parseException(sql1),
-      errorClass = "_LEGACY_ERROR_TEMP_0063",
-      parameters = Map("msg" -> value1),
+      errorClass = "INVALID_INTERVAL_FORMAT.INTERVAL_PARSING",
```

Comment:
The commit https://github.com/apache/spark/commit/0fd91c7805b8ad7e7f88665ecc59091d969499b5 renamed `errorClass` to `condition`. Please, rebase on the recent master and rename it.